nailgun/entities.py

```diff
@@ -21,8 +21,12 @@
 workings of entity classes.
 
 """
+# pylint:disable=no-member
```

Comment:
This is risky since it will silent no-member for the entire module, can you please comment why it would be added?